app/views/base.py

```diff
@@ -37,6 +37,96 @@ class BaseView(django.views.View):
         'lang': utils.strip,
     }
 
+    class Env(object):
+        """Class to store environment information used by views and templates.
+
+        Subclasses of BaseView may define their own Env class (which must be
```

Comment:
Can you also mention that, when a subclass defines its own Env class, it will automatically used to construct self.env in the subclass? That's the part I feel a bit tricky.